client/src/BookingManagment/actions/actions.js

```
@@ -0,0 +1,27 @@
+import { landings } from 'utils/api';
+
+import { updateLandings } from 'Booking/actions/actionsCreator';
+
+export const createLanding = landingData => dispatch => {
+    landings.createLanding(landingData)
+    .then(({ data }) => {
+        dispatch(updateLandings(data.landings));
+    })
+    .catch(() => {});
```

do not use empty catch, just don't use it if you don't need  it